client/me/help/help-contact/index.jsx

```diff
@@ -218,6 +222,33 @@ module.exports = React.createClass( {
 		widgetInput.onkeydown( { keyCode: KEY_ENTER } );
 	},
 
+	trackChatDisplayError: function() {
+		var { olark, isChatEnded } = this.state;
+
+		// If the user sent/received a message to/from an operator but the chatbox is not inlined
+		// then something must be going wrong. Lets record this event and give some details
+		if ( this.canShowChatbox() ) {
+			return;
+		}
+
+		const tracksData = {
```

Comment:
The `var` on line 226 should've been `const` and I use it when the variable content never changes